.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "unqlite"]
 	path = unqlite
-	url = git@github.com:symisc/unqlite.git
+	url = https://github.com/symisc/unqlite.git
```

Comment:
why is https more preferable over ssh?

Reply:
most github users are using https when clone which will result in an error

Reply:
agreed